### DISCUSSION TITLE
Name of attribute changed.

### DISCUSSION THREAD

**Discussion by han-alink** [Koenkk/zigbee2mqtt#8421] [Devices]
A multi sensor zigbee device is reporting temperature, humidity and pressure. If I look at the STATE, it is reporting a value for temperature_1 (and not for temperature):
{
    "humidity": 57.61,
    "linkquality": 72,
    "pressure": 10129,
    "temperature_1": 22.98
}
Who knows where this _1 is added to the attribute name?

**Comment by sjorge** [comment]
Which sensor is this?

IIRC there is a function that appends the endpoint name to the attribute if there are multiple endpoints, this might be the case here and if the endpoint has no name it uses the id.

**Comment by han-alink** [comment]
My own (diy).
-------- Oorspronkelijk bericht --------Van: Jorge Schrauwen ***@***.***> Datum: 19-08-21  10:36  (GMT+01:00) Aan: Koenkk/zigbee2mqtt ***@***.***> Cc: han-alink ***@***.***>, Author ***@***.***> Onderwerp: Re: [Koenkk/zigbee2mqtt] Name of attribute changed. (#8421) 
Which sensor is this?

**Reply by sjorge** [reply]
Depending on what converters you're using, e.g.
https://github.com/Koenkk/zigbee-herdsman-converters/blob/e765a3223ba6a7705b25976bd5f97ed60a29df3c/converters/fromZigbee.js#L316

postfixWithEndpointName() is probably adding the _1

**Reply by han-alink** [reply]
Thanks, a small change in my device converter solved the problem.
